src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenFunctionPointersTests.cs

```diff
@@ -11563,7 +11563,7 @@ class C { }
                 """;
 
             // https://github.com/dotnet/roslyn/issues/66187 tracks enabling runtime reflection support for this scenario.
-            var verifier = CompileAndVerify(source, symbolValidator: static module =>
+            var verifier = CompileAndVerify(source, options: TestOptions.UnsafeDebugDll, symbolValidator: static module =>
```

Comment:
@AlekseyTs However, there was a breaking change in this scenario. The breaking change doc seems sufficiently general to cover this scenario.